Odata-docs/client/v7/querying-odata-service.md

```diff
@@ -12,6 +12,15 @@ ms.topic: article
 
 **Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v7.md)]
 
+## Add an entity
+```csharp
+var context = new DefaultContainer(new Uri("https://services.odata.org/v4/TripPinServiceRW/"));
+    var person = Person.CreatePerson("russell", "Russell", "Whyte", new long());
+    context.AddToPeople(person);
+
+    context.SaveChanges();
```

Comment:
I tried this code and I got this error: https://github.com/smartstore/SmartStoreNET/issues/895. Do we need to capture information about how to solve this?

Reply:
Are you making an async call?

Reply:
Just tried both sync and async and still got the error

Reply:
#Resolved